apps/acrs/index.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
       behaviours: [SummaryPageBehaviour, CheckInformationGivenBehaviour],
       sections: require('./sections/summary-data-sections'),
       backLink: false,
-      journeyStart: '/who-is-completing-form'
+      journeyStart: '/who-completing-form'
```

Comment:
Please mention this fix when you squash

Reply:
Will do, although by then I hope it will have already been fixed by preceding PRs...but who knows.